Formula/xauth.rb

```diff
@@ -0,0 +1,34 @@
+class Xauth < Formula
+  desc "X.Org Applications: xauth"
+  homepage "https://www.x.org/"
+  url "https://www.x.org/pub/individual/app/xauth-1.1.tar.bz2"
+  sha256 "6d1dd1b79dd185107c5b0fdd22d1d791ad749ad6e288d0cdf80964c4ffa7530c"
+  license "MIT-open-group"
+  revision 1
+
+  depends_on "pkg-config" => :build
+  depends_on "util-macros" => :build
+  depends_on "libxmu" # brings in libxext, libx11, libxau
```

Comment:
I think we should depend on those explicitly if we need them.

Reply:
Agreed.  Your comment prompted me to check with `ldd` on Linux and `otool` on macOS to see if there were any other libraries it was linked to besides these 4.  On Linux, it also links to `libxcb` and `libxdmcp`, but not on macOS.  I've added those latter two as Linux only dependencies.